base/templates/base/base.html

```diff
@@ -1,129 +1,131 @@
+{% load static %}
+{% load bootstrap3 %}
 <!DOCTYPE html>
-<html>
-    
-    <head>
-        {% load bootstrap3 %}
-        {% bootstrap_css %}
-        {% bootstrap_javascript jquery=1%}
-        {% load staticfiles %}
-        {% block before-header %}{% endblock %}
-        <script type="text/javascript" src="{% static '/js/jsi18n'%}"></script>
-        <link rel="shortcut icon" href="{% static 'favicon.ico' %}"  type="image/x-icon" />
-        <title>
-            OEP{% block title %}{% endblock %}
-        </title>
-        <link rel="stylesheet" href="{% static 'css/theme-style.css' %}">
-        <link rel="stylesheet" href="{% static 'css/base-style.css' %}">
-        {% block after-header %}{% endblock %}
-    </head>
-
+<html lang="en">
+<head>
+    {% block before-head %}{% endblock before-head %}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
Can you describe why meta is useful here? (just for documenting the steps and to ease the review process of a file with many block of code changes)

Reply:
@Bachibouzouk, this is the minimal meta set, that nearly always should be implemented.

- `<meta charset="UTF-8">`: charset info
- `<meta name="viewport" content="width=device-width, initial-scale=1.0">`: viewport (import as responsive design base): https://www.w3schools.com/css/css_rwd_viewport.asp
- `<meta http-equiv="X-UA-Compatible" content="ie=edge">`: compatibility info for stating, that we only support the Microsoft edge browser (which soonish uses chromium as base :rocket:)